app/konnect/configure/runtime-manager/runtime-instances/troubleshoot.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Troubleshoot Runtime Instances
+no_version: true
+---
+
+## Out of sync runtime instance
+
+Occasionally, a {{site.base_gateway}} runtime instance might get out of sync
+with the {{site.konnect_short_name}} control plane. If this happens, you would
+see the status `Out of sync` on the Runtime Instances page, meaning the control
+plane can't communicate with the instance.
+
+Troubleshoot the issue using the following steps:
+
+1. Ensure the runtime instance is running. If it's not running, start it; if it
+is running, restart it.
+
+    Check the sync status in the Runtime Manager.
+
+1. Check the logs of the runtime that's appearing as `Out of sync`.
+{{site.base_gateway}} logs can be found at `<kong-prefix-directory>/logs`,
+where the default directory is `/usr/local/kong/logs`.
+
+    If you find either of the following:
+
+    * There is a connection error, where the instance failed to connect to the
+    control plane.
+    * Sending a ping to the control plane failed, or
+    the runtime instance failed to receive a ping response from the control plane.
+
+    Diagnose the host network where the instance resides.
```

Comment:
Maybe a more prescriptive sentence could work here: Since the sentence that introduces the types of errors is "if you find either of the following"

this sentence could be: 
"Then you could potentially have an issue on the network where the instance resides". "After resolving the network issue, restart the instance and check the sync status in the Runtime Manager.". 
And then there has to be a call to action in the case that they can't resolve the networking issue, or they do not have either of the two errors. Even if it is just filing a support ticket.